docs/pages/integrate.mdx

```diff
@@ -167,7 +167,57 @@ export function FarcasterEmbeds(props: { embeds: Array<Embed> }) {
   return (
     <div>
       {props.embeds.map((embed, i) => (
-        <RenderEmbed embed={embed} key={i} renderers={renderers} />
+        <RenderEmbed 
+          key={i} 
+          renderers={renderers} 
+          
+          // Required context
+          embed={embed} 
+          api={"https://api.modprotocol.org/api"}
+          
+          // Optional context
+          user={
+            id: "3",                // Current user's FID
```

Comment:
syntax error object notation. Also we should probably useMemo this so it doesn't rerender on every re-render. Although embeds should rarely have props or state changes